src/components/FilteredBy/index.tsx

```diff
@@ -0,0 +1,49 @@
+import { Typography } from '@mui/material'
+import { Box } from '@mui/system'
+
+export default function FilteredBy() {
+  const data = {
+    ['Address:']: '0x35500253DEB46fa8c2b271628c65DcF159206882',
+    ['Order ID:']: '45'
```

Comment:
usually good to have useMemo but no need here since will have to change this anyways